File: public/phase-2-20160620022203/C/dirty-stairs.c
// Name : Dirty Stairs
#include <stdio.h>
int main() {
	// input:
	// n :  number of steps in the staircase
	// m : number of dirty steps
	// d :  numbers of dirty steps 
	int n, m, d[3000], index;
	char result[105]="YES";
	scanf("%d %d", &n, &m);
	for(index = 0; index< m; index++) {
		scanf("%d", &d[index]);
	}


	// write your code here
	// store your results in `result`

	// output
	printf("%s", result);
	return 0;
}

File: public/phase-2-20160620022203/C_SHARP/dirty-stairs.cs
// Name : Dirty Stairs
using System;
namespace Application {
	class dirty_stairs {
		public static void Main(string[] args) {
		// input:
		// n :  number of steps in the staircase
		// m : number of dirty steps
		// d :  numbers of dirty steps 
		int index;
		string[] elements = (Console.ReadLine()).Split(' ');
		int n = Convert.ToInt32(elements[0]);
		int m = Convert.ToInt32(elements[1]);
		int[] d = new int[m];
		string[] d_elements = (Console.ReadLine()).Split(' ');
		for(index=0;index<m;index++)
			d[index] = Convert.ToInt32(d_elements[index]);


		// write your code here
		// store your results in `result`

		// output
		// Dummy Data
		string result="YES";

		Console.WriteLine(result);

		}	
	}
}